scalableminds/webknossos
# Persist tree visibility

### Remaining Todo

- [x] backend: implement updateTreeVisibility update action
- [x] frontend: send above action
- ~[ ] frontend: apply update tree group visibility action in skeleton.js~
- [x] frontend: clean up code (+ fix flow)
- [x] frontend: write tests? :S

### URL of deployed dev instance (used for testing):
- https://persisttreevisibility.webknossos.xyz

### Steps to test:
- open a skeleton (or hybrid) tracing
- create a few trees and groups
- toggle the visibility of some trees and groups
- save & reload
- the visibilities should be as before

### Issues:
- fixes #3503 

------
